karma.conf.js

```diff
@@ -2,8 +2,7 @@ const ci = !!process.env.CI;
 const watch = !!process.env.WATCH;
 const live = !!process.env.LIVE;
 
-const identifier = process.env.BROWSERSTACK_LOCAL_IDENTIFIER;
-const ip = process.env.IP_ADDR;
+const ip = 'bs-local.com';
```

Comment:
Why that ip?

Reply:
That is a domain that browserstack owns and knows how to forward to the device. `localhost` or the old way do not work any more